### ISSUE TITLE
Cannot run edgedb from a subpath of a project

### ISSUE THREAD

**Issue by vjjft** [edgedb/edgedb-cli#942]
Attempting to use the EdgeDB CLI from a project subpath results in conflicting error messages.

- Running `edgedb` from a subpath of an initialized project errors, with a message of:  ". . . project not initialized . . . Run `edgedb project init` . . ."
- Running `edgedb project init` from a subpath of a project errors, with a message of: "Found `edgedb.toml` . . . Project is already initialized."

Expected behavior:  Running `edgedb` from a subpath of an initialized project will succeed using the found `edgedb.toml` from the project path.

To reproduce, given an initialized edgedb project at `projectPath` and a valid `subPath`: 
```
$ cd <projectPath>/<subPath>

$ edgedb
edgedb error: ClientNoCredentialsError: project is not initialized and no connection options are specified
  Hint: Run `edgedb project init` or use any of `-H`, `-P`, `-I` arguments to specify connection parameters. See `--help` for details

$ edgedb project init
Found `edgedb.toml` in <projectPath>
Initializing project...
edgedb error: Project is already initialized.
```

- EdgeDB CLI Version: EdgeDB CLI 2.2.6+7eabbf9
- OS Version: macOS 13.0.1 (22A400)

**Comment by tailhook** [comment]
Okay, this is fixed in the main branch. You can try nightly:
```
curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh -s -- --nightly
```

I'll try to get bugfix released into stable next week.

**Comment by tailhook** [comment]
Fixed in 2.3.0
